functions/01_TotalActivation/Temporal_PFM/MyTemporal_pfm.m

```diff
@@ -10,25 +10,26 @@
 % to compute regularization coefficients)
 %
 % Outputs:
-% - TC_OUT is the n_time_points x n_ret_voxels 2D matrix of outputs from
+% - Activity_related is the n_time_points x n_ret_voxels 2D matrix of outputs from
 % the regularization step
 % - param is the updated structure with relevant parameters for TA; added
 % fields are 'LambdaTempFin' (vector with final regularization estimates
 % for each voxel), 'NoiseEstimateFin' (final noise estimate for each voxel)
 %
 % Implemented by Eneko Uruñuela, 13.12.2022
-function [TC_OUT, paramOUT] = MyTemporal_pfm(TCN, param)
+function [Activity_related,Activity_inducing,innovation, paramOUT] = MyTemporal_pfm(TCN, param)
 
     % The output from the algorithm (time x voxels) is initialized as
     % a matrix of zeros
-    TC_OUT = zeros(param.Dimension(4),param.NbrVoxels);
-
+    Activity_related = zeros(param.Dimension(4),param.NbrVoxels);
+    Activity_inducing = zeros(param.Dimension(4),param.NbrVoxels);
+    innovation = zeros(param.Dimension(4),param.NbrVoxels);
```

Comment:
Change TC_OUT for Activity_related and inicialise the new activity_inducing and innovation matrices